### DISCUSSION TITLE
Returning the last inserted Id

### DISCUSSION THREAD

**Discussion by JohnMwanyika** [prisma/prisma#17209] [Q&A] [answered]
## Problem

Why don't you guys have the support to return the inserted id when creating a new record

**Comment by pantharshit00** [comment]
Hello @JohnMwanyika,

We do return the whole document when you use the `create` method. Can you please paste the query where you are unable to extract the desired data?

**Reply by JohnMwanyika** [reply]
I want to be able to get the new inserted Id so I'd proceed to create milestones for the newly created project

**Reply by JohnMwanyika** [reply, marked as answer]
OK I figured it out thanks though,,just a small confusion in my naming conventions😄

**Reply by pantharshit00** [reply]
Okay! Can you confirm the original issue is solved now?

**Comment by JohnMwanyika** [comment]
<code>
const project = await prisma.project.create({
        data: {
          title: project_title,
          start_date: new Date(start_date),
          due_date: new Date(due_date),
          description: description,
          userUser_id: req.session.user.user_id,
        },
      });
let inserted_id = project.id;
console.log(project) //outputs an empty array
 res.json({ id: inserted_id }); //nothing is outputed

</code>
